.travis.yml

```diff
@@ -9,7 +9,9 @@ python:
   - "3.6"
 script:
   - pip install pipenv
+  - pipenv install
   - pipenv install --dev
+  - pipenv install nose2 pydocstyle pylint mypy coverage
```

Comment:
Why `pipenv install`, `pipenv install --dev` and `pipenv install SPECIFIC_PACKAGES`?

Reply:
Caught me. I'm still not 100% comfortable with pipenv. For some reason nose2 wasn't available even though it is specified in the dev section of the Pipfile. Will try cleaning that up.